pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/InvocationNode.java

```diff
@@ -49,11 +49,4 @@ default JMethodSig getMethodType() {
     }
 
 
-    /**
-     * Returns information about the overload selection for this call.
-     * Be aware, that selection might have failed ({@link OverloadSelectionResult#isFailed()}).
-     */
-    OverloadSelectionResult getOverloadSelectionInfo();
```

Comment:
Removing this method, wouldn't this be a breaking API change?

Reply:
I added it to a super interface, so I think it's not breaking anything :)